central/graphql/resolvers/image_components.go

```diff
@@ -450,3 +509,35 @@ func (resolver *imageComponentResolver) UnusedVarSink(ctx context.Context, args
 func (resolver *imageComponentResolver) FixedIn(_ context.Context) string {
 	return resolver.data.GetFixedBy()
 }
+
+func getImageCVEResolvers(ctx context.Context, root *Resolver, os string, vulns []*storage.EmbeddedVulnerability, query *v1.Query) ([]ImageVulnerabilityResolver, error) {
```

Comment:
Is there a problem here if sorting is involved?

Reply:
Also I wonder if we should make this function deterministic if sorting is not a requirement and sort to make the results deterministic

Reply:
Moved pagination into `getImageCVEResolvers` and added default sorting on ID if not sort cue in incoming query.